xbmc/utils/FileUtils.cpp

```diff
@@ -226,3 +227,61 @@ CDateTime CFileUtils::GetModificationDate(const std::string& strFileNameAndPath,
   }
   return dateAdded;
 }
+
+bool CFileUtils::ZebraListAccessCheck(const std::string &filePath)
```

Comment:
for starters std::begin and std::end are supported for plain arrays so using a `for const auto ptr: blacklist_files)` should work fine or `for (auto ptr = std::begin(blacklist_files); ptr != std::end(blacklist_files); ++ptr)` last I checked stupid android didn't support cbegin and cend so not sure about those.

could use std::array with std::string like